docs/source/release/v6.9.0/mantidworkbench.rst

```diff
@@ -54,27 +54,23 @@ New features
 
 Bugfixes
 ############
-- Fixed intermittent crash in the Instrument View caused by a project recovery save happening while the Instrument View was opening.
-- Fixed problem where spurious peaks can be added when picking single peaks in the Instrument View.
+- Fixed an intermittent crash caused by a project recovery save happening while the :ref:`Instrument Viewer <InstrumentViewer>` was opening.
+- Fixed a problem where spurious peaks could be added when picking single peaks.
 
 
 SliceViewer
 -----------
 
-New features
-############
-
-
 Bugfixes
 ############
-- Fixed intermittent error in Slice Viewer when reopening after a change in support for nonorthogonal axes.
-- Enables slice viewer's 'y' cut exports for event workspaces without giving errors in line plot and ROI modes.
-- Fixed bug in the cut viewer where the plot would not update after changing plot settings until the window had been resized.
-- Fixed layout bug when toggling the peaks overlays interface on/off.
-- Fix issue when adding peaks was not taking into account the projection matrix when calculating HKL.
-- Fixed a memory leak in the Slice Viewer colour bar.
-- Fixed spin box showing an incorrect value when a peak was selected that was outside the range of the data.
-- Fixed error in Slice Viewer when trying to click on the 2D plot after the `Add Peak` option is selected, but the peaks workspace has already been deleted.
+- Fixed an intermittent error when reopening SliceViewer after a change in support for non-orthogonal axes.
```

Comment:
Sorry I should have seen this last time, can we add a link to the Slice Viewer as well?